webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -234,8 +234,12 @@ private static PdfReader createPdf(String globalTitle, Date creationDate, Scramb
         }
 
         doc.open();
-        // Note that we ignore scrambleRequest.copies here.
-        addScrambles(docWriter, doc, scrambleRequest, globalTitle);
+        if (pdfFileName.equals("3x3x3 Fewest Moves Solution Sheet")) {
```

Comment:
I'm not very comfortable with this string comparison. I think the problem here is that there's a lot of boilerplate code involved in createing a PDF and it watermarking/closing it. Could we pull those out into 2 separate methods and then push the call to `addGenericFMCSolutionSheet()` up to `requestsToZip()`? If what I'm saying doesn't make sense, feel free to reach out to me on gchat and we can talk about this.